build.gradle

```diff
@@ -230,11 +230,14 @@ dependencies {
     compile group: 'org.apache.logging.log4j', name: 'log4j-core', version: '2.13.0'
     compile group: 'org.apache.commons', name: 'commons-lang3', version: '3.9'
     compile group: 'commons-io', name: 'commons-io', version: '2.3'
+    compile 'org.projectlombok:lombok:1.18.12'
     implementation 'io.grpc:grpc-netty-shaded:1.25.0'
     implementation 'io.grpc:grpc-protobuf:1.25.0'
     implementation 'io.grpc:grpc-stub:1.25.0'
     implementation 'javax.annotation:javax.annotation-api:1.3.2'
 
+    annotationProcessor 'org.projectlombok:lombok:1.18.12'
```

Comment:
We discussed that we will remove this new dependency, right ?

Reply:
Fully removed in the latest PR